Frends.MicrosoftSQL.ExecuteQuery/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.1] - 2023-01-27
```

Comment:
Property name has changed, so there will be changes from user point of view, so I think you should increase minor version instead of build version.